x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/action/TransportStopTransformAction.java

```diff
@@ -458,4 +474,34 @@ private void waitForTransformStopped(
             listener.onFailure(e);
         }));
     }
+
+    private ActionListener<Response> cancelTransformTasksWithNoAssignment(
+        final ActionListener<Response> finalListener,
+        final TransformNodeAssignments transformNodeAssignments
+    ) {
+        final ActionListener<Response> doExecuteListener;
+        doExecuteListener = ActionListener.wrap(response -> {
```

Comment:
Could you get rid of `doExecuteListener` and just call `return ActionListener.wrap(...)`?

Reply:
good points, not sure, why I ended up with this, maybe due to a re-factoring. Thanks for the hint.